python/pyspark/sql/types.py

```diff
@@ -641,6 +642,50 @@ def __eq__(self, other):
         return type(self) == type(other)
 
 
+class PointUDT(UserDefinedType):
```

Comment:
It looks like the original `ExamplePointUDT` is still in the tests file, too.  Why not just use that?